client/src/components/overview/DefaultView.jsx

```diff
@@ -1,29 +1,76 @@
-import React, { useContext, useEffect, useCallback } from 'react';
+import React, { useContext, useState, useEffect, useCallback } from 'react';
+import ExpandedView from './ExpandedView.jsx';
+import ModalForm from '../shared/Modal.jsx';
+import { getProductStyles } from '../../request.js';
+import Selectors from './selectors.jsx';
+import ProductDetails from './productDetails.jsx';
 import { OverviewContext } from './OverviewContext.jsx';
 import { getProductPhotosOfAStyle } from '../../request.js';
+import { ProductDetailsContainer } from '../styles/Overview/ProductDetails.styled.js';
 import { DefaultViewContainer, BackgroundImageContainer, BackgroundImage, ThumbnailSliderContainer } from '../styles/Overview/DefaultView.styled';
 import { Arrow, FadedArrow, ActiveThumbnail, FadedThumbnail, ThumbnailContainer, SliderContainer } from '../styles/Overview/Slider.styled';
 
-export const DefaultView = () => {
+export const DefaultView = (props) => {
+  const [ styles, setStyles ] = useState([]);
+  const [ selectedStyle, setSelectedStyle ] = useState(props.styles[0].style_id);
+  const [ expandedView, setExpandedView ] = useState(false);
   const { images, setImages, currentImage, handleKeyPress } = useContext(OverviewContext);
   const context = useContext(OverviewContext);
 
-  useEffect(() => {
-    getProductPhotosOfAStyle(39333, 234004)
-      .then(results =>
-      setImages(results.photos.map((photo, id) => {return {id, ...photo}}))
-    );
+  useEffect( () => {
+    getProductPhotosOfAStyle(props.productId, selectedStyle)
+      .then( (results) => {
+        setImages(results.photos.map((photo, id) => {
+          return {id, ...photo}}
+        ));
+      })
+      .catch( (err) => {
+        console.error(err);
+      });
   }, []);
 
+  const handleImageClick = () => {
+    setExpandedView(true);
+  }
+
+  const updateStyle = (styleId) => {
+    setSelectedStyle(styleId);
+    getProductPhotosOfAStyle(props.productId, styleId)
+      .then( (results) => {
+        setImages(results.photos.map((photo, id) => { return {id, ...photo}} ));
+      })
+      .catch( (err) => {
+        console.error(err);
+      });
+  }
```

Comment:
Instead of doing this, you can have the `useEffect` hook on lines 20-30 listen for changes in the `selectedStyle`. To keep it DRY